frontend/src/hooks/crud/useInfiniteFind.ts

```diff
@@ -35,7 +35,8 @@ export const useInfiniteFind = <
       string[],
       Error,
       string[],
-      [QueryType, EntityType, any]
+      TBasic[],
+      [QueryType, EntityType, string]
```

Comment:
How to test

`frontend/src/app-components/inputs/AutoCompleteEntitySelect.tsx`

Before the change (main)

<img width="723" height="207" alt="image" src="https://github.com/user-attachments/assets/b5545cf8-309a-42c7-b97d-4b1510dd1360" />

After the change (1227-issue---usefind-hook-querykey-items-are-not-serialized)
<img width="723" height="207" alt="image" src="https://github.com/user-attachments/assets/0a640949-798b-41c7-937f-0e6aee4f6a1e" />